code/modules/client/preferences/sounds.dm

```diff
@@ -28,23 +28,23 @@
 	savefile_key = "sound_instruments"
 	savefile_identifier = PREFERENCE_PLAYER
 
-/datum/preference/toggle/sound_tts
+/datum/preference/choiced/sound_tts
 	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
 	savefile_key = "sound_tts"
 	savefile_identifier = PREFERENCE_PLAYER
 
-/datum/preference/toggle/sound_tts_blips
-	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
-	savefile_key = "sound_tts_blips"
-	savefile_identifier = PREFERENCE_PLAYER
-	default_value = FALSE
+/datum/preference/choiced/sound_tts/init_possible_values()
+	return list(TTS_SOUND_ENABLED, TTS_SOUND_BLIPS, TTS_SOUND_OFF)
+
+/datum/preference/choiced/sound_tts/create_default_value()
+	return TTS_SOUND_ENABLED
 
 /datum/preference/numeric/sound_tts_volume
 	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
 	savefile_key = "sound_tts_volume"
 	savefile_identifier = PREFERENCE_PLAYER
 
-	minimum = 0
+	minimum = 1
```

Comment:
Not a fan of this at all. The intent as a user of setting TTS volume to 0 is obvious, shouldn't have to hunt for a different setting for it.

Reply:
It was more so players would have to actually disable it so that way we can more easily tell who has the pref enabled or not

Reply:
We can already easily tell that